newmarkov.py

```diff
@@ -0,0 +1,60 @@
+import json
+import io
+import random
+import numpy
```

Comment:
Really Curious At What NUMPY is used for.

Reply:
it's been a while since anyone's worked on this, but iirc it was basically used to select randomly from a list. I decided it would be easier to just add duplicates of a word to the list of possible words after a given word than do some other, infinitely more well thought out, solution.